test/unittests/CandidateTest.php

```diff
@@ -211,7 +237,8 @@ public function testSetDataWithArraySucceeds()
 
     /**
      * Test setData method sets data when passing variable name and value
-     *
+     * TODO Same test as above? 
```

Comment:
This test seems to be the same as the one above it. Could someone take a look?

Reply:
Looks the same to me too

Reply:
@kongtiaowang ?

Reply:
It is the same!

Reply:
@AlexandraLivadas are you planning on removing this duplicate?